### PR TITLE
Epic/managing student profile

### PR DESCRIPTION
**Pull request by arcvelit**
Merging Kevin's code with main

### CODE REVIEW

**Review by Razvan-Alexandru — CHANGES_REQUESTED** [review]
Make sure your branch is up to date with main before merging. There are a lot of changes that should not be here

File: app/src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
+import './Header.css';
 
 export default function Header() {
   return (
-    <div class="header">
-        <div class="logo">
+    <div className="header">
+        <div className="logo">
             <h2>Hired</h2>
-            <svg class="logo-image" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"/></svg>
```
[the remark this answers is not in the document]
Comment (Amir8Abdullah) [reply app/src/components/Header/Header.jsx:8]: All changes from class to className are nothing. In HTML you use class and in Javascript you use className. They are the exact same thing, but this change needs to be done to make it functional.

File: app/src/App.tsx [hunk 2]
```diff
@@ -7,16 +7,18 @@ import './components/Header/Header.css';
 import LoginPage from "./LoginPage/LoginPage";
 import './LoginPage/LoginPage.css';
 
-import Posts from "./components/posts/Posts";
+import Posts from "../src/components/posts/Posts"
```
[the remark this answers is not in the document]
Comment (KD54321) [reply app/src/App.tsx]: resolved

**Comment by wam82** [comment]
This pull request is outdated and should no longer be opened

**Pull request closed without merging**